src/main/java/edu/cornell/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -34,7 +36,7 @@
 
 
 public class ProcurementCardCreateDocumentServiceImpl extends org.kuali.kfs.fp.batch.service.impl.ProcurementCardCreateDocumentServiceImpl {

```

Comment:
This class's "createProcurementCardDocument" method had a small amount of formatting and loop iteration changes for the 07/19 patch. Is it worth updating our implementation of the method at this time?

Reply:
updated